4.1/crud-filters.md

```diff
@@ -224,9 +224,7 @@ $this->crud->addFilter([
       4 => 'Not available',
     ];
 }, function($values) { // if the filter is active
-    // foreach (json_decode($values) as $key => $value) {
-    //     $this->crud->addClause('where', 'published', $value);
-    // }
+    // $this->crud->addClause('whereIn', 'published', json_decode($values));
```

Comment:
```suggestion
    // $this->crud->addClause('whereIn', 'status', json_decode($values));
```

For coherence the column name should be the same as of the filter. Not mandatory, but easier to understand for those who read.